### PR TITLE
Introduce auto land in arc diff create template

### PR DESCRIPTION
**Pull request by txchen**

### CODE REVIEW

**Comment by ptarjan** [comment]
I see `getDefaultCreateFields` returns a `$notes`. Is that what is similar semantically to what we want?

Also, can you please show us what it looks like in the diff description for our users?

**Comment by txchen** [comment]
```
just test

Summary:

Test Plan:

Reviewers:

Subscribers:

Task Link:

# Uncomment next line if you\'d like this diff to land when tests pass (http://go/ready_to_land)
# Tags: #ready_to_land
#
# NEW DIFFERENTIAL REVISION
# Describe the changes in this new revision.
#
# Included commits in branch test3:
#
#         d44aca00b474 just test
#
# arc could not identify any existing revision in your working copy.
# If you intended to update an existing revision, use:
#
#   $ arc diff --update <revision>
```

**Comment by txchen** [comment]
$notes is always empty, unless it is parsing from commit message. Which means for create, it is always empty.

Line 1409, also assume $notes is from update, not create. So I don't think we should use that, to make the change minimal.

File: src/workflow/ArcanistDiffWorkflow.php [hunk 1]
```diff
@@ -1445,6 +1445,11 @@ private function getCommitMessageFromUser() {
     }
 
     $issues = array_merge(
+      array(
+        pht('Uncomment next line if you\'d like this diff to land when tests pass (http://go/ready_to_land)'),
```
Comment (ptarjan) [inline src/workflow/ArcanistDiffWorkflow.php:1449]:
```suggestion
        pht('Uncomment the next line if you\'d like this diff to land when tests pass and it is approved (http://go/ready_to_land)'),
```

**Review by ptarjan — APPROVED** [review]
Is this too long? Maybe @alexeagle has some wordsmithing. And if it is too long does `vim` wrap it?

File: src/workflow/ArcanistDiffWorkflow.php (hunk 1 above)
Comment (txchen) [reply src/workflow/ArcanistDiffWorkflow.php:1449]: What's our company rule of line width? (I hope it is not 80) I personally use 120, and not using wrap for vim.

**Comment by txchen** [comment]
Is this better?

```
      array(
        pht('Phabricator auto land: (http://go/ready_to_land)'),
        pht('Uncomment the next line to auto land when tests pass and it is accepted.'),
        pht('Tags: #ready_to_land'),
        '',
      ),
```

**Review by alexeagle — APPROVED** [review]

**Pull request merged**